yaas/utils.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+def remove_hrefs(el):
```

Comment:
The hrefs are the most irritating part about the Ambari API. You can't take a blueprint from `cluster?format=blueprint` and feed it back to the Ambari server because of hrefs. Same with `repo`. Once you remove the hrefs, they're valid again.

My understanding is that the hrefs are there to indicate what parts of the API you can query for more information (e.g. each host in `/api/v1/host` has an href). That's not particularly useful to us, as we're not really exposing the raw output of the API. Note that when `--raw` is specified you will see the hrefs.